game/src/layer/map.rs

```diff
@@ -219,7 +219,7 @@ impl Static {
         let edits = app.primary.map.get_edits();
         for r in &edits.changed_roads {
             let r = app.primary.map.get_r(*r);
-            let orig = EditRoad::get_orig_from_osm(r);
+            let orig = EditRoad::get_orig_from_osm(r, app.primary.map.get_config().driving_side);
```

Comment:
God help us when we get to mapping the hong-kong / mainland china border.

Reply:
![image](https://user-images.githubusercontent.com/217057/92263852-29137b80-ee92-11ea-80f4-58507554c188.png)

Reply:
It'll actually be easy, just refactor this into a function that maps a way to left/right! The cars will wiggle around each other correctly